### DISCUSSION TITLE
Data import

### DISCUSSION THREAD

**Discussion by yaredb** [tech-acs/chimera-docs#9] [Bug report] [answered]
it overwrites the questionnaire tables and I have to change them back every time I import.

**Comment by amestsantim** [comment]
Perhaps we can add a switch to the import command to tell it that it's not an empty database.
Is the questionnaire table that is losing data?

**Reply by baakal** [reply]
This issue affect production, after introducing a new indicator we export the data using php artisan chimera:data-export, in our development environment the questionnaire connection is set for local database, when we import it to production it overwrites the existing connection.
the expected behavior is when the questionnaire has a record with that particular name, it should ignore it and not update it.

**Comment by amestsantim** [comment, marked as answer]
On the chimera:data-export command, there is now a --exclude-table option which can be used to exclude one or more tables from the data-export. The initial dump will likely not use this but subsequent "update" exports can include --exclude-table=questionnaires and avoid including the questionnaires table in the export (unless there are new questionnaires added/modified). 
The option also accepts multiple values which means you can exclude multiple values like so:
```
php artisan chimera:data-export --exclude-table=questionnaires --exclude-table=permissions
```
